build/scripts/update-from.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Copyright (c) 2019-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# update FROM lines in Dockerfiles to the latest base images
+
+# get script
+if [[ ! -f /tmp/updateBaseImages.sh ]]; then 
+    pushd /tmp >/dev/null || exit
+    curl -sSLO https://raw.githubusercontent.com/redhat-developer/codeready-workspaces/master/product/updateBaseImages.sh
```

Comment:
Hi, FYI Che should not depend on codeready workspaces

Also using master is not nice as it can break later

Reply:
well we could move this script to another repo if you'd like, and then have CRW depend on it. Maybe put it in che-machine-exec?

Reply:
which branch would be better?

Reply:
psst. where would you like this script moved/copied?